setup.py

```diff
@@ -16,15 +16,48 @@
 with open('CHANGES.rst') as history_file:
     history = history_file.read()
 
-requirements = ['Click>=6.0', ]
+tests_require = [
+    'check-manifest>=0.35',
+    'coverage>=4.0',
+    'isort>=4.2.15',
+    'mock>=1.3.0',
+    'pydocstyle>=1.0.0',
+    'pytest-cov>=1.8.0',
+    'pytest-random-order>=0.5.4',
+    "pytest-pep8>=1.0.6",
+    'pytest>=3.8.1,<4',
+]
 
-setup_requirements = ['pytest-runner', ]
+extras_require = {
+    'docs': [
+        'Sphinx>=1.5.6,<1.6',
+        'invenio-accounts>=1.0.0',
+    ],
+    'tests': tests_require,
+}
+
+extras_require['all'] = []
+for name, reqs in extras_require.items():
+    extras_require['all'].extend(reqs)
+
+setup_requires = [
+    'pytest-runner>=2.6.2',
+]
+
+install_requires = [
+    'Flask>=0.11.1',
+    # "requests" has hard version range dependency on "idna" and "urllib3"
+    # Every time "idna" and "urllib3" are updated, installation breaks because
+    # "requests" dependencies are not resolved properly.
+    'urllib3<1.25,>=1.21.1',  # from "requests"
+    'idna>=2.5,<2.8',  # from "requests"
+]
 
-test_requirements = ['pytest', ]
+packages = find_packages()
```

Comment:
This can be skipped actually, since below there's a `packages=find_packages(include=['autobot']),` line